bin/dcps_tests.lst

```diff
@@ -332,6 +332,7 @@ tests/DCPS/SubscriberCycle/run_test.pl: !DCPS_MIN !DDS_NO_CONTENT_FILTERED_TOPIC
 tests/transport/rtps/run_test.pl: !DCPS_MIN RTPS
 tests/transport/rtps_reliability/run_test.pl: !DCPS_MIN RTPS
 tests/transport/spdp/run_test.pl: !DCPS_MIN RTPS
+tests/transport/rtps_direct_write/run_test.pl: !DCPS_MIN RTPS
```

Comment:
```suggestion
tests/transport/rtps_directed_write/run_test.pl: !DCPS_MIN RTPS
```
Let's make it "directed write" for consistency with the spec.  That way someone can search the repository and find this test.